packages/mc-scripts/src/utils/credentials-storage.ts

```diff
@@ -1,22 +1,22 @@
-import fs from 'fs';
-import os from 'os';
-import path from 'path';
+import fs from 'node:fs';
+import os from 'node:os';
+import path from 'node:path';
 import type { TMcCliAuthToken, TMcCliCredentialsStorage } from '../types';
 import doesFileExist from '../utils/does-file-exist';
 
-const homedir = os.homedir();
-const credentialsFolderPath = path.join(homedir, `.commercetools`);
-const credentialsFilePath = path.join(
-  credentialsFolderPath,
-  'mc-credentials.json'
-);
-
 class CredentialsStorage {
-  static location = credentialsFilePath;
+  private credentialsFilePath: string;
```

Comment:
Not sure what the issue is, but somehow the mocks weren't working correctly when using the static `location` property